### PR TITLE
Admin test: first click into the element before typing

### PR DESCRIPTION
**Pull request by coolo**
This makes it more robust with real browsers

@Martchus

### CODE REVIEW

File: t/ui/13-admin.t [hunk 1]
```diff
@@ -347,19 +347,22 @@ subtest 'job property editor' => sub() {
...
+        $ele->click();
+        $ele->send_keys('500');
+        $driver->find_element_by_id('editor-description')->send_keys('Test group');
+        $driver->find_element_by_css('p.buttons button.btn-primary')->click();
         # ensure there is no race condition, even though the page is reloaded
         t::ui::PhantomTest::wait_for_ajax;
 
-        # now reload the page to see if we succeeded
-        $driver->get($driver->get_current_url());
+        $driver->refresh();
```
Comment (Martchus) [inline t/ui/13-admin.t]: If you are wondering why we used `$driver->get($driver->get_current_url());` before: refresh() just didn't work.
But apparently it works now, so updating Selenium was definitely a good idea.

File: t/ui/14-dashboard.t [hunk 2]
```diff
@@ -114,11 +115,14 @@ is(scalar @{$driver->find_elements('h2', 'css')}, 2, 'a single, empty group para
...
-    $driver->find_element('#filter-time-limit-days',      'css')->send_keys('2');            # appended to default '14'
-    $driver->find_element('#filter-form button',          'css')->click();
+    $driver->find_element_by_css('#filter-panel .panel-heading')->click();
+    $driver->find_element_by_id('filter-group')->send_keys('SLE 12 SP2');
+    my $ele = $driver->find_element_by_id('filter-limit-builds');
+    $ele->send_keys(KEYS->{end}, '8');    # append to 3
+    $ele = $driver->find_element_by_id('filter-time-limit-days');
+    $ele->click();
+    $ele->send_keys(KEYS->{end}, '2');    # appended to default '14'
+    $driver->find_element_by_css('#filter-form button')->click();
```
Comment (Martchus) [inline t/ui/14-dashboard.t]: Could be `find_element_by_id` now, too.

**Comment by Martchus** [comment]
So this prevents the strange behaviour of the admin test? I can verify tomorrow, but LGTM.

File: t/ui/14-dashboard.t (hunk 2 above)
Comment (coolo) [inline t/ui/14-dashboard.t]: ids don't have spaces

**Pull request merged**